src/rest_documents_GET.ecpp

```diff
@@ -32,11 +32,12 @@
 #include <dirent.h>
 #include <unistd.h>
 
+#include <cxxtools/jsondeserializer.h>
 #include <tnt/httperror.h>
 
 #include "helpers.h"
 
-#include "fty_security_wallet_library.h"
+#include "fty_security_wallet.h"
```

Comment:
This link is not necessary in my opinion. The others include was more explicit.

Reply:
AFAIK <library_name>_library.h include file is considered internal/helper by zproject and outside libraries should link with <library_name>.h. @EldoreiJK , what do you think?

Reply:
I would even say that we should not use those one it this particular case but "secw_producer_accessor.h" and "secw_exception.h"

Reply:
Fixed